### DISCUSSION TITLE
Welcome to Optiplex-3020-Hackintosh-OpenCore Discussions!

### DISCUSSION THREAD

**Discussion by varszegimarcell** [varszegimarcell/Optiplex-3020-Hackintosh-OpenCore#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by squalo81** [comment]
hi guys i'm having a problem with my dell 3020 big sur, i managed to install the problem without problems big sur following your guide, my dell is a 3020 with graphics 4600 16gb of ram, using it with logic x i have problems of memory in particular with plugins, I was wondering having set the nvram with grub to 64 there is no way to increase this memory with a variable?
